examples/appSpec/zomes/three/code/index.ts

```diff
@@ -13,7 +13,7 @@ import {
 
 /*----------  Public Functions  ----------*/
 
-@zome_function
+@capability('main')
```

Comment:
Dept. of Names Dept. Q: does it make more sense to describe each of these functions as a "capability" or a "zome function"? I like that calling the decorator `capability` makes it clear what the argument should be, but using `@zomeFunction(capabilityName)` makes more sense to me with regard to what it actually is decorating.

Reply:
Willem and I were just chatting, and his latest thought was this...
```
@capability('agent')
namespace main {
...
}
```

where multiple zome functions are defined within the namespace. `agent` refers to the access membrane